src/memory/client/MemoryBrowser.tsx

```diff
@@ -106,6 +137,7 @@ export class MemoryBrowser extends React.Component<Props, State> {
           args: {
             address: this.addressReq,
             length: parseInt(this.lengthReq),
+            child: this.childReq,
```

Comment:
It doesn't hurt to include this when sending to cdt-gdb-adapter/Memory, but continue to ensure that it isn't necessary to have so that the memory code works directly with cdt-gdb-adapter.

Reply:
I have updated it